extra-web/palemoon/autobuild/patch

```diff
@@ -0,0 +1,7 @@
+abinfo "Moving platform folder..."
+rm -rv "$SRCDIR/platform"
+mv -v "$SRCDIR/../uxp/" "$SRCDIR/platform"
+
```

Comment:
Remove this routine, and rename autobuild/patch to autobuild/prepare. Let Autobuild3 handle patching.

Reply:
> Remove this routine, and rename autobuild/patch to autobuild/prepare. Let Autobuild3 handle patching.

Files involved in patching are under `uxp`, which should be moved to `$SRCDIR/platform` directory before ab3 handles it.

Or consider revert to the old `GITSRC` method?

Reply:
Just use Git. If it's Internet-related issue, then use a Relay server.